client/my-sites/comments/main.jsx

```diff
@@ -10,21 +10,33 @@ import { localize } from 'i18n-calypso';
 import Main from 'components/main';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import DocumentHead from 'components/data/document-head';
+import CommentList from './comment-list';
+import CommentNavigation from './comment-navigation';
 
 export class CommentsManagement extends Component {
 
 	static propTypes = {
 		basePath: PropTypes.string,
+		siteSlug: PropTypes.string,
+		status: PropTypes.string,
 		translate: PropTypes.func,
 	};
 
 	render() {
-		const { translate, basePath } = this.props;
+		const {
+			basePath,
+			siteSlug,
+			status,
+			translate,
+		} = this.props;
 		return (
-			<Main className="comments">
+			<Main className="comments" wideLayout>
 				<PageViewTracker path={ basePath } title="Manage Comments" />
 				<DocumentHead title={ translate( 'Manage Comments' ) } />
-				<div>Hello World!</div>
+				<div className="comments__primary">
+					<CommentNavigation siteSlug={ siteSlug } status={ status } />
+					<CommentList />
```

Comment:
Thinking about it, having both `CommentNavigation` and the actual list of comment sounds the most sensible choice.
At the same time, though, I'm afraid that eventually the list part might end up needing some methods of its own, and we'd have to move it into its own component anyway.

Also: yes, IDs would be great! I'll see what I can do with those!